submission-history/src/index.ts

```diff
@@ -0,0 +1,281 @@
+// Required libs
+import express from 'express';
+import mysql from 'promise-mysql';
+var fs = require('fs');
+var multer = require('multer');
+var path = require("path");
+const bodyParser = require("body-parser");
+const server = express();
+var SqlString = require('sqlstring'); // To prevent SQL injection
+
+// Use libs
+server.use(express.json());
+server.use(express.urlencoded({ extended: true })); 
+server.use(bodyParser.text()); // for req.body
+server.use(bodyParser.urlencoded({ extended: true })); // for req.body
+// Using EJS as view engine
+server.set('view engine', 'ejs');
+
+// Needed Variables
+// Gerry
+let challengeID = [];
+let challengeName = [];
+let dateSubmittedG = [];
+let didAllTestsPass = [];
+
+// Austin
+let userName = [];
+let userId = [];
+let dateSubmittedA = [];
```

Comment:
What are all these global variables for? Using global variables is very likely to break things when the server is handling multiple concurrent requests.

Reply:
Remove those global variables